src/stcal/ramp_fitting/constants.py

```diff
@@ -0,0 +1,5 @@
+DO_NOT_USE = 2**0           # Bad pixel. Do not use.
+SATURATED = 2**1            # Pixel saturated during exposure.
+JUMP_DET = 2**2             # Jump detected during exposure
+NO_GAIN_VALUE = 2**19       # Gain cannot be measured
+UNRELIABLE_SLOPE = 2**24    # Slope variance large (i.e., noisy pixel)
```

Comment:
We are going to want to figure out a way to make sure that JWST and Roman use the same DQ mnemonics, and a way to formalize that in a test across both pipelines.  Otherwise the shared code model will fall apart.